### PR TITLE
East 2020 Event

### PR DESCRIPTION
**Pull request by dshanske**
Add East 2020 Event.

### CODE REVIEW

File: public/east.php [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+      <h1 class="ui inverted header p-name">
+        <?= $title ?>
+      </h1>
+
+      <h2><?= $date ?></h2>
+
+      <div class="alert">
+        <h2>Online only</h2>
+        <p>IndieWebcamp East is 100% online during Pacific Daylight Time hours on the 14th and 15th of November 2020.</p>
```
Comment (tantek) [inline public/east.php]: s/Pacific Daylight Time/Eastern Standard Time

**Review by tantek — CHANGES_REQUESTED** [review]
Looks pretty good. Just one detail (PDT vs EST) for now.

**Review by dshanske — COMMENTED** [review]
Again

**Review by tantek — APPROVED** [review]
LGTM

**Pull request merged**